src/tools/compiletest/src/header.rs

```diff
@@ -1490,29 +1490,38 @@ fn ignore_llvm(config: &Config, line: &str) -> IgnoreDecision {
         }
     }
     if let Some(actual_version) = config.llvm_version {
-        if let Some(rest) = line.strip_prefix("min-llvm-version:").map(str::trim) {
-            let min_version = extract_llvm_version(rest).unwrap();
-            // Ignore if actual version is smaller the minimum required
-            // version
+        if let Some(value) = config.parse_name_value_directive(line, "min-llvm-version") {
+            let value = value.trim();
```

Comment:
Remark: Separate from this PR, we should just make the parse functions trim automatically. I can't imagine a legitimate use-case for directives actually wanting to see untrimmed whitespace.

Reply:
I didn't want to touch that in the current directive handling, tbh. I prefer we just redo this entirely.